src/stylesheets/core/_variables.scss

```diff
@@ -1,96 +1,121 @@
-// Typography
-$em-base:             10px;
-$base-font-size:      rem(17px);
-$small-font-size:     rem(14px);
-$lead-font-size:      rem(20px);
-$title-font-size:     rem(52px);
-$h1-font-size:        rem(40px);
-$h2-font-size:        rem(30px);
-$h3-font-size:        rem(20px);
-$h4-font-size:        rem(17px);
-$h5-font-size:        rem(15px);
-$h6-font-size:        rem(13px);
-$base-line-height:    1.5;
-$heading-line-height: 1.3;
-$lead-line-height:    1.7;
-
-$font-sans:           'Source Sans Pro', $helvetica;
-$font-serif:          'Merriweather', $georgia;
-
-$font-normal:         400;
-$font-bold:           700;
+// These variables drive almost every aspect of the U.S. Web Design
+// system. Every variable is declared as !default so that it can be
+// overridden by a previous import.
```

Comment:
I've got an example of this in the new `README.md` changes [here](https://github.com/18F/web-design-standards/blob/28a608222e79780184c3bb7e0d39f75848a195df/README.md#sass). Once this gets merged in, I'll update here to point to that example. ^5